### PR TITLE
Colors in result

### PR DESCRIPTION
**Pull request by tomas-knap**
connects to #32 
connects to #27

### CODE REVIEW

File: src/templates/createnewtask/controller.js [hunk 1]
```diff
@@ -14,7 +14,11 @@
         // Files
         $scope.fileProvision = settings.fileProvisionTypes[0];
         $scope.files = {};
-
+        //Supported knowledge bases
```
Comment (tomas-knap) [inline src/templates/createnewtask/controller.js]: This is acceptable in some test, but not in code, which was denoted as solving #27. Discuss with @brodecva  how to manage reviews and when to close issues.

File: src/templates/createnewtask/controller.js [hunk 2]
```diff
@@ -14,7 +14,11 @@
...
+        $scope.availableKBs = ["DBpedia"];
+        //TODO smazat
```
Comment (tomas-knap) [inline src/templates/createnewtask/controller.js]: This is acceptable in some test (hard coding certain variables), but not in code, which was denoted as solving #27. Discuss with @brodecva how to manage reviews and when to close issues.

**Comment by tomas-knap** [comment]
Minor issue: When running odalic on top of books.csv and clicking on the class for canadian writters, I got: 


screen shot 2016-08-08 at 16 41 50

Solution: If the combo box is too long, such situation must be properly handled

**Comment by brodecva** [comment]
> Minor issue: When running odalic on top of books.csv and clicking on the class for canadian writters, I got:

We have noticed that too. I will create separate issue still.

**Comment by brodecva** [comment]
> Solving #32, #27

I thought about this and... I have no actual problems with these pull request issues. I think they do not duplicate the original business issues as long as they are used as intended by Github - discussing the pull request branch (see https://help.github.com/articles/using-pull-requests/#pull-request-discussion ).

For other things (mainly tracking progress, general debates, research discussions,...), the original business issues are more suitable.

**Comment by brodecva** [comment]
> We have noticed that too. I will create separate issue still, but this minor problem does not block this pull request.

Confirmed fixed.

**Comment by brodecva** [comment]
Unfortunately sending of the feedback is yet again broken! The client sents feedback for a task named "undefined" and therefore it is not recognized by the server. Apart from that, everything seems fine. Please fix ASAP.

**Comment by brodecva** [comment]
> Unfortunately sending of the feedback is yet again broken! The client sents feedback for a task named "undefined" and therefore it is not recognized by the server. Apart from that, everything seems fine. Please fix, @KataBoku , ASAP.

Hm, tried again and it works now. Taken back.

**Comment by brodecva** [comment]
I will merge the result myself this time, because @istvan-satmari does not come back until tonight and Tomas has already seen the code.

**Pull request merged**